tests/functional/export.bats

```diff
@@ -0,0 +1,167 @@
+    local workdir="$FIXTURES"/workdir
+    local suite="$FIXTURES"/suite.json
+
+    rm -rf "$WORKDIR" "$STORE"
+
+    # This is needed to be able to run inside mock, as it uses some temp files
+    # and that is not seamlesly reachable from out of the chroot by
+    # libvirt/kvm
+    export BATS_TMPDIR BATS_TEST_DIRNAME
+    export LIBGUESTFS_DEBUG=1 LIBGUESTFS_TRACE=1
+    cd "$FIXTURES"
+
+    helpers.run_ok "$LAGOCLI" \
+        init \
+        --template-repo-path "$REPO_CONF" \
+        --template-repo-name "$REPO_NAME" \
+        --template-store "$STORE" \
+        "$suite"
+
+    # This is needed to be able to run sudo inside the chroot
+    echo 'Defaults:root !requiretty' > /etc/sudoers.d/lago_functional_tests
+}
+
+@test "export.start: start everything at once" {
+    common.is_initialized "$WORKDIR" || skip "workdir not initiated"
+    cd "$FIXTURES"
+    helpers.run_ok "$LAGOCLI" start
+}
+
+@test "export.place_dummy_file" {
+    cd $FIXTURES
+    rm -rf dummy_file
+
+    local content="$(date)"
+    echo "$content" > "dummy_file"
+
+    helpers.run_ok "$LAGOCLI" \
+        copy-to-vm \
+        "$VM_NAME" \
+        dummy_file \
+        /root/dummy_file_inside
+
+    # Just to be sure that the file was written to disk
+    sleep 2
+}
+
+@test "export.fail_to_export_running_env" {
+    cd "$FIXTURES"
+
+    helpers.run_nook "$LAGOCLI" \
+        "export" \
+        --standalone \
+        --dst-dir "$STAND_ALONE_EXPORT_DIR"
+}
+
+@test "export.stop" {
+    cd "$FIXTURES"
+
+    "$LAGOCLI" shell "$VM_NAME" "cd /sbin && ./poweroff"
+    sleep 5
+    common.is_initialized "$WORKDIR" || skip "workdir not initiated"
+    helpers.run_ok "$LAGOCLI" stop
+}
+
+@test "export.export_stand_alone: stand alone images export" {
+    cd "$FIXTURES"
+
+    helpers.run_ok common.is_stopped $WORKDIR
+    helpers.run_ok "$LAGOCLI" \
+        "export" \
+        --standalone \
+        --dst-dir "$STAND_ALONE_EXPORT_DIR"
+}
+
+@test "export.export_layred: layred images export" {
+    cd "$FIXTURES"
+
+    helpers.run_ok common.is_stopped $WORKDIR
+    helpers.run_ok "$LAGOCLI" \
+        "export" \
+        --dst-dir "$LAYERED_EXPORT_DIR"
+}
+
+@test "export.remove_base_env" {
+    cd "$FIXTURES"
+
+    helpers.run_ok "$LAGOCLI" destroy --yes
+}
+
+@test "export.init_exported_sa_env" {
+    cd "$STAND_ALONE_EXPORT_DIR"
+
+    local suite="${FIXTURES}/exported_suite.json"
+    # will be used by exported_suite.json
+    export EXPORTED_ENV=$STAND_ALONE_EXPORT_DIR
+
+    helpers.run_ok "$LAGOCLI" \
+        init \
+        --template-repo-path "$REPO_CONF" \
+        --template-repo-name "$REPO_NAME" \
+        --template-store "$STORE" \
+        "$suite"
+}
+
+@test "export.test_exported_env: testing standalone export" {
+    cd $FIXTURES
+    local dummy_file_content=$(cat dummy_file)
+    cd "$STAND_ALONE_EXPORT_DIR"
+
+    helpers.run_ok "$LAGOCLI" start
+    dummy_file_inside_content=$("$LAGOCLI" shell "$VM_NAME" "cat /root/dummy_file_inside")
```

Comment:
not directly related - but just emphasises the fact we need to fix https://github.com/lago-project/lago/issues/409 as all of these sort of checks might fail too.
btw one way to overcome this is to generate the dummy file with N lines and compare only its 'tail -N'